src/main/__tests__/authOptions.spec.ts

```diff
@@ -1,13 +1,13 @@
-import { prepareAuthOptions } from '../authOptions'
+import { computeAuthOptions } from '../authOptions'
```

Comment:
I don't see why `computeAuthOptions` is better than `prepareAuthOptions`.

Reply:
To me `prepareAuthOptions` means precreating it, so return an instance of `AuthOptions` while `computeAuthOptions` sounds to me building sth else from an instance of `AuthOptions` (here it's `AuthParams` though).